modules/ROOT/pages/variable-transformer-reference.adoc

```diff
@@ -19,7 +19,7 @@ Set Variable provides a way to set the name and value of Mule variable, along wi
 
 | Variable Name (`variableName`)
 | Required
-| Name of the variable, which can be a string or a DataWeave expression. Variable names can include only numbers, characters, and underscores. For example, hyphens are not allowed in the name.
+| Name of the variable, which can be a string. Variable names can include only numbers, characters, and underscores. For example, hyphens are not allowed in the name.
```

Comment:
```suggestion
| Name of the variable, which must be a string. Variable names can include only numbers, characters, and underscores. For example, hyphens are not allowed in the name.
```
Otherwise it seems as if there was another possibility.

Additionally, maybe we should make it explicit that expressions are not allowed. Because someone may come saying that expressions are also strings...